packages/melody-idom/src/dom_util.ts

```diff
@@ -69,6 +69,28 @@ const getActiveElement = function(node) {
     return isDocumentRoot(root) ? root.activeElement : null;
 };
 
+/**
+ * Function that checks whether some element is contained inside a node. This function
+ * has a fallback implementation for cases where Node.prototype.contains is
+ * not implemented (e.g. IE11 for SVGElements).
+ * @param {!Node} node The node that should be parent of elm.
+ * @param {?Element} elm The element that should be child of elm.
+ * @return {Boolean} whether or not elm is contained within node.
+ */
+const nodeContainsElm = function(node, elm) {
+    if (node.contains) {
+        return node.contains(elm);
+    }
+
+    do {
+        if (elm === node) {
+            return true;
+        }
+    } while ((elm = elm && elm.parentNode));
```

Comment:
Does prettier accept an extra pair of parenthesis around `elm && elm.parentNode`? Would aid with readability quite a bit.

Otherwise you could consider switching to an implementation like this:
```
while (elm !== node && elm) {
    elm = elm.parentNode;
}
return elm === node;
```

Not sure if it works but it might be a bit smaller.

Reply:
No prettier does not accept the extra brackets, I can change the implementation to the one you suggested, tested it and it works